observability/otel/otel-collector/otel-collector-connectors.adoc

```diff
@@ -94,6 +53,126 @@ service:
 # ...
 ----
 
+[id="routing-connector_{context}"]
+== Routing Connector
+
+The Routing Connector routes logs, metrics, and traces to specified pipelines according to resource attributes and their routing conditions, which are written as OpenTelemetry Transformation Language (OTTL) statements.
+
+:FeatureName: The Routing Connector
+include::snippets/technology-preview.adoc[]
+
+.OpenTelemetry Collector custom resource with an enabled Routing Connector
+[source,yaml]
+----
+  config: |
+    connectors:
+      routing:
+        table:
+          - statement: route() where attributes["X-Tenant"] == "dev" # <1>
+            pipelines: [traces/dev] # <2>
+          - statement: route() where attributes["X-Tenant"] == "prod"
+            pipelines: [traces/prod]
+        default_pipelines: [traces/dev] # <3>
+        error_mode: ignore
+        match_once: false
+    service:
+      pipelines:
+        traces/in:
+          receivers: [otlp]
+          exporters: [routing]
+        traces/dev:
+          receivers: [routing]
+          exporters: [otlp/dev]
+        traces/prod:
+          receivers: [routing]
+          exporters: [otlp/prod]
+----
+<1> Routing conditions written as OTTL statements.
+<2> Destination pipelines for routing the matching telemetry data. In this case, the `traces/dev` pipeline when the `X-Tenant` attribute is equal to `dev`.
+<3> In this example, if `X-Tenant` attribute is not `dev` or `prod`, the telemetry data will be forwarded to those pipelines.
+
+[NOTE]
+====
+The connector routes exclusively based on OTTL statements, which are limited to resource attributes. Currently, it does not support matching against context values.
+====
+
+.Parameters used by the Routing Connector
+[options="header"]
+[cols="a,a,a"]
+|===
+|Parameter |Description |Default
+
+|`table`
+|Connector routing table containing routing conditions and destination pipelines.
+|`[]`
```

Comment:
Should this be N/A as its a required field. ?

Reply:
The default is `[]`. If you don't set anything, it will return the error `invalid routing table: the routing table is empty`.

I would say it is obvious you need to provide a table, but I would do it in the description instead of saying something in the default value column.